scripts/project-trigger.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+
+json_file="../projects/projects.json"
+gh_token=$GH_TOKEN
+git_ref="main"
+workflow_organization_name="cncf-tags"
+workflow_project_name="green-reviews-tooling"
+workflow_dispatcher_file_name="benchmark-pipeline.yaml"
+
+if [ -z "$gh_token" ]; then
+    echo "[ERR] GH_TOKEN not set"
```

Comment:
```suggestion
    echo "GH_TOKEN not set"
```

Some output is important but lets keep it as simple as possible. 

We don't need these prefixes so pls remove them from all the echo statements. If its an error the echo text should explain that.